app/scripts/lib/multichain/address.test.ts

```diff
@@ -32,11 +32,10 @@ const NON_EVM_ADDRESSES = [
   {
     address: 'bc1ql49ydapnjafl5t2cp9zqpjwe6pdgmxy98859v2',
   },
-];
+] as TestAddress[];
```

Comment:
should the `TestAddress` type be updated to reflect what is actually in `ETH_ADDRESSES`, instead of casting here?